packages/dd-trace/test/appsec/index.express.plugin.spec.js

```diff
@@ -0,0 +1,123 @@
+'use strict'
+
+const axios = require('axios')
+const getPort = require('get-port')
+const path = require('path')
+const os = require('os')
+const fs = require('fs')
+const agent = require('../plugins/agent')
+const appsec = require('../../src/appsec')
+const Config = require('../../src/config')
+
+withVersions('body-parser', 'body-parser', version => {
```

Comment:
I think in this test we should pin `express` version instead of `body-parser` one since it is testing `query-string` based blocking